### ISSUE TITLE
updated_at isn't set on instance creation

### ISSUE THREAD

**Issue by jimrollenhagen** [jagi/meteor-astronomy-timestamp-behavior#1]
I think it makes sense to set updated_at to the current time on `events.beforeInsert`. Many frameworks implement this behavior this way, and it seems better for application usage to avoid doing null checks on this field, etc.

Thoughts?

**Comment by jimrollenhagen** [comment]
I'm happy to do the work to implement this, FWIW

**Comment by lukejagodzinski** [comment]
Yes, you are right, I forgot about this. Feel free to create PR :) Thanks!
